src/main/java/github/grace000/erversay/ResponseBuilder.java

```diff
@@ -0,0 +1,37 @@
+package github.grace000.erversay;
+
+import java.util.HashMap;
+
+import static github.grace000.erversay.Constants.*;
+
+public class ResponseBuilder {
+    private String code = "200";
```

Comment:
it seems like the constants/here could be refactored so 200 isn't duped.. like the default code = 200 and then it's referenced here?

Reply:
Thanks!